teaser/logic/buildingobjects/useconditions.py

```diff
@@ -123,21 +125,28 @@ class UseConditions(object):
         AixLib: Used for internal gains profile on top-level
         Annex: Not used (see Annex examples)
     min_ahu: float [m3/(m2*h)]
-        Zone specific minimum specific air flow supplied by the AHU
+        Zone specific minimum specific air flow supplied by the AHU.
+        Air flow varies lineary with the persons_profile,
+        person=0 -> min_ahu, person=1 -> max_ahu.
```

Comment:
please double check with AixLib and export

Reply:
Please see my latest changes, and confirm after you´ve double-checked it.